src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -122,5 +122,15 @@ namespace AppInstaller::CLI::Workflow
         }
 
         context.Reporter.Info() << Resource::String::UninstallFlowUninstallSuccess << std::endl;
+    }
+    
+    void GetDependencies(Execution::Context& context)
+    {
+        const auto& installer = context.Get<Execution::Data::Installer>();

```

Comment:
There is no `Installer` for `uninstall`.  I commented previously on how this would need to be done.